mathesar/tests/api/test_mathesar_types_api.py

```diff
@@ -0,0 +1,59 @@
+from mathesar.api.display_options import DISPLAY_OPTIONS_BY_TYPE_IDENTIFIER
```

Comment:
Usually we name the test files similarly to the file they're testing, I'd consider renaming this to `ui_types` or putting this in `tests/api/ui`. Please make this change in a future PR, though, since I don't want to block merge.